### DISCUSSION TITLE
vim打开终端，必须同时按ctrl加backspace键才能进行退格？可否设置成不需要按ctrl键，直接backspace键就能进行退格

### DISCUSSION THREAD

**Discussion by wxzvim** [ayamir/nvimdots#922] [Q&A]
如下图：
![image](https://github.com/ayamir/nvimdots/assets/110797126/2d574d37-4db4-467b-ac59-0e4a17bcfbd4)
否则每次按ctrl键，感觉好麻烦

**Comment by Jint-lzxy** [comment]
This is probably an issue specific to your terminal emulator. Neovim processes the keycodes it receives as-is, so nothing can be done on the editor side. Make sure `stty -a` gives you terminal-compatible `cchars` and your `terminfo` description is compatible with the `unibilium` library you link Neovim against.

**Comment by Jint-lzxy** [comment]
Closed due to inactivity. Please reactivate if further support is needed.
